cranelift/assembler/meta/src/instructions/or.rs

```diff
@@ -0,0 +1,32 @@
+use crate::dsl::{fmt, inst, r, rex, rw, sxl, sxq, Features::*, Inst, LegacyPrefixes::*, Location::*};
+
+pub fn list() -> Vec<Inst> {
+    /*
+    In 64-bit mode, r/m8 can not be encoded to access the following byte registers if a REX prefix is used: AH, BH, CH, DH
+    So following REX instructions are unimplemented
+    [REX+80 /1 ib] [OR r/m8, imm8]
+    [REX+08/r] [OR r/m8, r8]
+    [REX+OA/r] [OR r8, r/m8]
```

Comment:
I see this implemented down below?

Reply:
E.g., isn't `REX+08 /r` implemented by line 23 below?

Reply:
To recap a chat Andrew and I had offline - These instructions are in fact implemented, so I'm removing this block comment. The confusion arose due to, for `REX+08 /r` and the likes, I had thought the `w` bit needs to be set, similar to what is done `REX.W+0D id` where we explicitly set `w()`.
So, for instructions which are `REX+..` and not `REX.W+..` the w() being set is not required.
Now, since IA manual states
```
 In 64-bit mode, r/m8 can not be encoded to access the following byte registers if a REX prefix is used: AH, BH, CH, DH
```
and there isn't a way currently to differentiate between if REX is used or not, the logic to check if the above regs are used is not implemented here. 
I let the fuzzer run for a long time to see if capstone would fail if one of these prohibited regs is used. On closer inspection, it looks like the high regs `ah, bh, ch ,dh` are not emitted by cranelift at all (for `or`, `and` etc.), so capstone doesn't fail.
I looked at [this file](https://github.com/abrown/wasmtime/blob/2b3fe80b65c4943428ec1d587bf35740cd977401/cranelift/codegen/src/isa/x64/inst/emit.rs#L608) in cranelift for some reference. I can only see lower 8 bit regs emitted like `al, bl, cl, dl`.